deploy/kubernetes/Jenkinsfile.yml

```diff
@@ -12,11 +12,6 @@ when:
     tags: true                 ## for all the tags
 platform: "immutable && ubuntu-22" ## default label for all the stages
 stages:
-    checks:
```

Comment:
For some reason these checks were not migrated in https://github.com/elastic/beats/pull/32723